dto/scenario_iterations.go

```diff
@@ -31,19 +31,22 @@ type ScenarioIterationBodyDto struct {
 	Rules                         []RuleDto `json:"rules"`
 	ScoreReviewThreshold_deprec   *int      `json:"scoreReviewThreshold"` //nolint:tagliatelle
 	ScoreReviewThreshold          *int      `json:"score_review_threshold"`
-	ScoreRejectThreshold_deprec   *int      `json:"scoreRejectThreshold"` //nolint:tagliatelle
-	ScoreRejectThreshold          *int      `json:"score_reject_threshold"`
-	Schedule                      string    `json:"schedule"`
+	ScoreBlockAndReviewThreshold  *int      `json:"score_block_and_review_threshold"`
+	ScoreRejectThreshold_deprec   *int      `json:"scoreRejectThreshold"`   //nolint:tagliatelle
+	ScoreDeclineThreshold         *int      `json:"score_reject_threshold"` //nolint:tagliatelle
+	// TODO: rename it, after the previous migration is finished
```

Comment:
Comment probably at the wron place

Reply:
hmm yes kind of, at first I had it after the noling pragma, but that moved it all to the next line and broke it.
Anyway, I'm going to touch this again very soon and I'll clean it up

Reply:
In fact: score_reject_threshold is sort of the new "wrong dto value", I think I'll migrate the dto to use score_decline_threshold, but in a separate PR where I'll also remove the ones already marked as deprecated

Reply:
what I won't do now is rename the field in DB